cetav-ddw/lalibertad-cetav
# Cursos list

## Description

Adding courseList component in the home page and the tecnicos page

### How to verify this PR

1. npm run dev
2. go to pages and verify that is there


inicio
![CETAV - Google Chrome 1_11_2023 16_11_29](https://github.com/cetav-ddw/lalibertad-cetav/assets/106450222/46bd10ce-e83b-4ea9-b7d2-ba66d9ab6892)


tecnicos pages
![CETAV - Google Chrome 1_11_2023 16_13_28](https://github.com/cetav-ddw/lalibertad-cetav/assets/106450222/4d1c8793-ceb5-4aad-a594-bd271a73a7e6)
